mrmustard/lab_dev/circuits.py

```diff
@@ -287,7 +287,7 @@ def serialize(self, filestem: str = None):
     @classmethod
     def deserialize(cls, data: dict) -> Circuit:
         r"""Deserialize a Circuit."""
-        comps, path = map(data.pop, ("components", "path"))
+        comps, path = data.pop("components"), data.pop("path")
```

Comment:
I don't understand why this change in needed.  Aren't the two equivalent.  The changed one is probably more readable.

Reply:
yep, readability